[2022-02-08 08:16:27,954] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.2 2022-02-08T08:07:59.998874+00:00 [None]>
[2022-02-08 08:16:27,996] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.2 2022-02-08T08:07:59.998874+00:00 [None]>
[2022-02-08 08:16:27,997] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 08:16:27,997] {taskinstance.py:1088} INFO - Starting attempt 1 of 4
[2022-02-08 08:16:27,997] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 08:16:28,019] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): 2> on 2022-02-08T08:07:59.998874+00:00
[2022-02-08 08:16:28,025] {standard_task_runner.py:52} INFO - Started process 603 to run task
[2022-02-08 08:16:28,032] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hello_world', '2', '2022-02-08T08:07:59.998874+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmpsh1p8bkr', '--error-file', '/tmp/tmp0drwuher']
[2022-02-08 08:16:28,032] {standard_task_runner.py:77} INFO - Job 7: Subtask 2
[2022-02-08 08:16:28,115] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.2 2022-02-08T08:07:59.998874+00:00 [running]> on host 8c4baac7362d
[2022-02-08 08:16:28,251] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=2
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T08:07:59.998874+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-08T08:07:59.998874+00:00
[2022-02-08 08:16:28,253] {logging_mixin.py:104} INFO - Yeaaahh
[2022-02-08 08:16:28,253] {python.py:151} INFO - Done. Returned value was: None
[2022-02-08 08:16:28,286] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=2, execution_date=20220208T080759, start_date=20220208T081627, end_date=20220208T081628
[2022-02-08 08:16:28,347] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-08 08:16:28,364] {local_task_job.py:151} INFO - Task exited with return code 0
[2022-02-08 08:16:28,413] {dagrun.py:444} INFO - Marking run <DagRun hello_world @ 2022-02-08 08:07:59.998874+00:00: manual__2022-02-08T08:07:59.998874+00:00, externally triggered: True> successful
